patches/minecraft/net/minecraft/tileentity/TileEntityFurnace.java.patch

```diff
@@ -1,6 +1,40 @@
 --- before/net/minecraft/tileentity/TileEntityFurnace.java
 +++ after/net/minecraft/tileentity/TileEntityFurnace.java
-@@ -37,20 +37,18 @@
+@@ -3,6 +3,7 @@
+ import net.minecraft.block.Block;
```

Comment:
Use the full class name, don't patch the import
This is a waste of text.

Reply:
Removed